### DISCUSSION TITLE
Plans for 1.2.0?

### DISCUSSION THREAD

**Discussion by szpak** [gradle-nexus/publish-plugin#177] [Ideas]
We have a number of things done & fixed [since 1.1.0](https://github.com/gradle-nexus/publish-plugin/milestone/3). It might be good to release it "soon". What important element we should wait for with 1.2.0?

https://github.com/gradle-nexus/publish-plugin/pull/145 - ? - or it could wait for the next release @vlsi?

The next "bigger" version (not counting patch releases with bugfixes) would be 1.3.0 or [2.0.0](https://github.com/gradle-nexus/publish-plugin/milestone/5) (depending on the compatibility changes with the required Java & Gradle versions or changes in the API).

**Comment by vlsi** [comment]
I think #145 is ready for merge.
I've added tests, and improved error messages.

I believe the next "big" change (of course, it goes after the release) should be to stop using `allprojects` and stop asking for "root project only".
Currently, GNSP is incompatible with precompiled script plugins, which is sad.

See https://github.com/gradle-nexus/publish-plugin/issues/84#issuecomment-986943076 , https://github.com/gradle-nexus/publish-plugin/issues/81

**Comment by TWiStErRob** [comment]
I wouldn't hold off on anything to get the next version released, be it 1.1.1 or 1.2.0.
The last release was almost 2 years ago.

Version numbers are free, there's nothing holding you to actually introducing some major non-breaking change in a minor version.

Any "future" work can go into a next minor or major release as you noted.
We especially need a published version asap, if you're planning to drop Java 8 / Kotlin 1.3/4 / Gradle 5/6 compatibility.

**Comment by szpak** [comment]
1.2.0 has been just released. Let's move #145 to the next release - 1.3.0 or 2.0.0 (depending on its backward compatibility).
